packages/expo-firebase-analytics/src/Analytics.ts

```diff
@@ -0,0 +1,199 @@
+import { UnavailabilityError } from '@unimodules/core';
+
+import ExpoFirebaseAnalytics from './ExpoFirebaseAnalytics';
+
+import parseConfig from './parseConfig';
+
+if (!ExpoFirebaseAnalytics) {
+  console.warn(
+    "No native ExpoFirebaseAnalytics module found, are you sure the expo-firebase-analytics's module is linked properly?"
+  );
+}
+
+type GoogleServicesConfig = { [key: string]: any };
+
+/**
+ * Get the bundled Google Services config file.
+ * This is useful for debugging if your app was built properly.
+ */
+export function getBundledGoogleServicesConfig(): null | GoogleServicesConfig {
+  return ExpoFirebaseAnalytics.app || null;
+}
+
+/**
+ * Similar to `firebase.initializeApp()` on web but works to start a native Firebase app while the app is running.
+ * This can be used to test the native iOS Firebase app in the Expo client.
+ * This method should not be used in production, instead the app should be bundled with the native Google Services files via the `app.json`.
+ *
+ * @param googleServices Platform specific Google Services file for starting a Firebase app during runtime
+ */
+export async function initializeAppDangerously(
```

Comment:
should be moved to core

Reply:
I was actually thinking of moving the "app-management" bits to `expo-firebase-app`. That would then also provide an interface for dealing with multi-app  environments.

Do you think that initializing the "default" native firebase is necessary, or more a nice thing to have? I would prefer to provide a single initialisation path using the googleServicesFile setting.

Reply:
how would that work in managed apps @IjzerenHein? would we initialize automatically from manifest on app load?

Reply:
Yep, that's what `expo-firebase-core` currently does

Reply:
This way your JS app code will also be identical for the Expo client and stand-alone builds, bare-apps